src/main/java/seedu/tinner/model/role/ReminderDate.java

```diff
@@ -53,35 +60,46 @@ public static boolean isValidDeadline(String test) {
     }
 
     /**
-     * Returns true if the deadline is within a week from the current {@code LocalDateTime}.
+     * Returns true if the reminder date is within a week from the current {@code LocalDateTime}
+     * or if the date is null.
      *
      * @return true if is within a week from the current date.
      */
     public boolean isWithinReminderWindow() {
+        if (this.value == null) {
+            return false;
+        }
         LocalDateTime oneWeekAway = LocalDateTime.now().plusDays(reminderWindow);
```

Comment:
Spotted a mini bug of my own here. I forgot to change the LocalDateTime variable name to oneReminderWindowAway. Could you help to just make a quick change to this for me so I don't need to make a whole PR just for that one variable 😅 . Thanks!

Reply:
Sure no worries